yarn.lock

```diff
@@ -19,6 +19,23 @@
   optionalDependencies:
     chokidar "^2.1.8"
 
+"@babel/cli@^7.6.2":
```

Comment:
Are all these yarn.lock changes intentional?

Reply:
Yes, they were not committed with the original gatsby-plugin PR